.github/workflows/deploy-dotorg.yml

```diff
@@ -0,0 +1,21 @@
+name: Deploy to WordPress.org
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    name: New Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: WordPress plugin deploy
+        uses: 10up/action-wordpress-plugin-deploy@stable
+        env:
+          SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+          SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+      - name: Upload release assets
+        uses: softprops/action-gh-release@v0.1.14
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: ${{github.workspace}}/${{ github.event.repository.name }}.zip
```

Comment:
I think we should probably use a custom "ZIP build" workflow, since there's a bunch of things in the GitHub repository that shouldn't be in the plugin ZIP. At a minimum, I think we should have a `.gitattributes` file (or `.distignore`) that ensures the ZIP only includes what we need in a production ZIP (e.g. no tests, no docs, no development tooling etc.). See also https://github.com/10up/action-wordpress-plugin-deploy#sample-baseline-files

Reply:
Added a `.distignore` file https://github.com/WordPress/performance/blob/add/42-actions-automatic-deployment-wordpressdotorg/.distignore